script/prerender/fleet.ts

```diff
@@ -98,7 +171,7 @@ async function createWorkersStack() {
 
   console.log(`Creating ${workersStackName} stack...`);
 
-  await cfnClient.send(new CreateStackCommand({
+  await sendWithRetries(cfnClient, new CreateStackCommand({
```

Comment:
I tried using `callWithRetries` with `cfnClient.send.bind(cfnClient)` but typescript always got mad at some type deep within the AWS libraries, probably because `send` returns different types depending on the argument.